### PR TITLE
Newsweek Clone done using Bootstrap Classes.

### PR DESCRIPTION
**Pull request by Abhigyan001**
- We used Bootstrap classes to clone the newsweek.com website.
- Navigation bar was created with d-flex property and was hidden for smaller screen devices using d-none class.
- The main three columns were divided into grids using row and col bootstrap classes and for responsiveness, md, sm, lg, and xl classes were used.
- The sticky top classes were applied for columns 2nd and 3rd to make it fixed while column 1 was scrolling.
- The use of media screen properties was kept as minimum as possible.

### CODE REVIEW

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team :wave: 
Status: Changes Required :recycle: 

## :star:  IMPRESSIVE BOOTSTRAP USAGE :+1: 

Great job so far :+1:  :star: 
Let's cover a few details to make your project better :arrow_down: 

## Navbar

- [x]  Please fix the overflowing links at the given width.

- [x]  Also you have a horizontal scroll at the same point.

**Screenshot from the project :camera_flash:**
navbreaks

## Main

- [x]  Images on the opinion section should be better looking. Please fix them and make them appear at the same size, just like the original website. Also, it has overflow in the second image.
**Screenshot from the project :camera_flash:**
opinions

- [x]  More stories section: Please fix the images, they should be in proper size.

**Screenshot from the project :camera_flash:**
opinion2

**Screenshot from the project :camera_flash:**
more stories

- [x]  We can make sponsored insight fit better at this scale.

**Screenshot from the project :camera_flash:**
sponsored insight

- [x]  "World" section images should fully cover the page width at this scale.

**Screenshot from the project :camera_flash:**
world
 
## That's it. After doing the changes, you can request another code review :blush: 

Thanks for your time and efforts... :star: 
With all my respect... :pray:

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Hello there! Kevin here. First of all, you @Abhigyan001 and @Nkuria did a great job on making the previously required changes! Secondly I must say that your page meets most of the requirements  👏  , However, we have some minor issues happening and both of you have to fix them before getting this awesome project approved. Check below ⤵

Explanation of requests to make your page more similar to the original one can be found [here](https://user-images.githubusercontent.com/56762529/92530630-205cd580-f203-11ea-8a4e-06a6d7d54e78.png) 👀

## [REQUIRED CHANGES]
### Clone REQUIREMENTS 👥 (Those adjusts to make your page look like the real one)⤵
#### Medium screen issues
- [x] Hey team, those images (Marianne and Noah) are a bit bigger than it should, kindly improve them.
- [x] OPTIONAL: Remove that blank space at the fixed navbar (It only have blank space when its at the top of the page in large screens)
#### Yours
image
#### Original
image
- [ ] Please fix that overlapping
#### Yours
image
#### Original
image

#### Small screen issues
- [ ] OPTIONAL: Remove that blank space at the fixed navbar (It only have blank space when its at the top of the page in large  screens)
- [ ] Please fix that overlapping
- [ ] Fix the size of those images (Marianne and Noah)
#### Yours
image

#### Original
image

## [OPTIONAL CHANGES]
- [x] Add a favicon to make it more similar to the original page;
- [x] Put img, css and js folder in a assets folder. (To make your repo more professional)
#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵

**⚠These ⤵ are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
- [ ] Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)  (e.g. ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright)
Explanation: Your page have 0 `<article>` and `<section>` tags, but it definitively should have a lot of them.

#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
### Large screen issues
- [x] Your fixed navbar doesn't look like the original which is more thiner, with the logo at the leftmost part and without the weather and date
#### Yours
image
#### Original
image

#### Yours:
- [x] That button is breaked (don't fill the whole space as it shoiu;ld) as you can see.
image
#### Original:
image

## Summary
✨ Your CSS looks very clean and professional! 👨‍💻
✨ Nice usage of the bootstrap!
✨ Overall your page looks really nice, congrats! 👌
✨ Your repo is very organized with img, css and js folders (tho, you also can improve it more by following my optional changes) 🗃
## Please make the required changes and resubmit for a new code review once you are ready💪

### @Abhigyan001 and @Nkuria It was a pleasure to review your clean and organized code, hope to see you both more often! Keep up that amazing work! 🥇


## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_button: 
### HELLO AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT. THERE ARE SOME REMAINING CHANGES TO BE DONE KINDLY CHECK BELOW :arrow_down: 

- [ ] Kindly you need to center your website well. If you see below :arrow_down:  The body of your website is not properly centered 
body

- [ ] At mobile view there is an overflow in your page kindly fix this. 

over flow

- [ ] At the mobile view there are no images in the more stories section. Kindly You need to add the images. Check below :arrow_down: 

CLONE
more stories clone

ORIGINAL
more stories original

- [ ] Great work improving the opinion section. But there are still some remaining changes to be done especially at mobile view kindly check below :arrow_down: 

### CLONE
opinion clone

### ORIGINAL
opinion original

- [ ] On the news section just near the footer, there are some remaining improvements to be done kindly check below. Compare them to the original and kindly make them look like the original

### CLONE
NEWS LETTER  SEC CLONE
### ORIGINAL
NEWS SEC ORIGINAL

### great progress lets make it even better :muscle: 

### IF YOU HAVE ANY QUESTION KINDLY REACH ME ON SLACK @PETERROBERT

### HAPPY CODING :computer:

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.grey {
+  background-color: #f4f5f6;
+}
+
+.btn-color {
+  background-color: #ff0500;
+  color: white;
+}
```
Comment (andresporras3423) [inline css/styles.css]: Group with selector on line 74

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  }
+
+  .mobiletext {
+    text-align: center !important;
+    padding-right: 2% !important;
+  }
+
+  .mobilepad {
+    padding-left: 2% !important;
+    margin-left: 13% !important;
```
Comment (andresporras3423) [inline css/styles.css]: Don´t overuse important

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Abhigyan Mahanta & Nick Kuria** 
Good job following the required changes by the previous TSE :muscle:  :muscle:
There are still some improvements to do before you can go ahead:
- [x] Don't overuse important
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.
- [x]  In small screen size, align the elements of the footer as the original:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] In small screen size, on the subscribe section, modify the width of the image and move the text to be in front of the image:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] In the medium screen size, "THE DEBATE" section, align the text to the left:

| Yours | Original |
|:---:|:---:|
|image | image |

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: css/styles.css (hunk 1 above)
Comment (MuriloRoque) [reply css/styles.css]: - [x] You still didn't group this selector, as requested previously.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @Abhigyan001 and @peterrobert :wave: 

### Thank you for making the changes to the footer, subscribe section, and "The Debate" sections, as requested by the previous TSE, but you still didn't fix the !important overuse and grouped the selectors appointed by him. Please check my comments in your code below.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello again @Abhigyan001 and @peterrobert :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**